Program.cs

```diff
@@ -8,24 +13,38 @@ public static async Task Main()
             var config = new ServiceConfig("TestConfig.env");
 
             // Create the client
-            var client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            IRepositoryApiClient client;
 
-            // Get a list of repository names
+            if (config.AuthorizationType.Equals("LfdsUsernamePassword", StringComparison.OrdinalIgnoreCase))
+            {
+                client = RepositoryApiClient.CreateFromLfdsUsernamePassword(config.Username, config.Password, config.Organization, config.RepositoryId, config.BaseUrl);
+            }
+            else if (config.AuthorizationType.Equals("AccessKey", StringComparison.OrdinalIgnoreCase))
+            {
+                client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            }
+            else
+            {
+                Trace.TraceWarning("Invalid value for authorization type.");

```

Comment:
Would be better if you used the actual name of the environment variable `AUTHORIZATION_TYPE` and also list out  the valid values like `AccessKey` and `LfdsUsernamePassword`